app/javascript/react/index.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
-import { Provider } from 'react-redux';
-import { BrowserRouter as Router } from 'react-router-dom';
+import { ConnectedRouter } from 'connected-react-router';
+import { connect } from 'react-redux';
+import { Spinner } from 'patternfly-react';
+import PropTypes from 'prop-types';
 import Routes from './config/Routes';
-import store from '../redux';
 import NotificationList from './screens/App/common/NotificationList';
+import createReducers from '../redux/reducers';
 
 // todo: convert this to use ManageIQ.redux.addReducer, ManageIQ.redux.store
 // <Provider store={store}>{componentRegistry.markup(component, data, store)}</Provider>,
-export default () => (
-  <Provider store={store}>
-    <Router>
-      <React.Fragment>
-        <NotificationList />
-        <Routes store={store} />
-      </React.Fragment>
-    </Router>
-  </Provider>
-);
+class OwerView extends React.Component {
```

Comment:
did you mean to call this "Overview" or something else? I guess since this is the root of the application (and not technically the Overview), maybe we can just call this class "Index" ? just a consideration...